userd/userd.go

```diff
@@ -71,17 +50,29 @@ func (ud *Userd) Init() error {
 		return err
 	}
 
-	reply, err := ud.conn.RequestName(busName, dbus.NameFlagDoNotQueue)
-	if err != nil {
-		return err
+	ud.dbusIfaces = []dbusInterface{
+		&Launcher{},
 	}
+	for _, iface := range ud.dbusIfaces {
+		reply, err := ud.conn.RequestName(iface.Name(), dbus.NameFlagDoNotQueue)
+		if err != nil {
+			return err
+		}
 
-	if reply != dbus.RequestNameReplyPrimaryOwner {
-		err = fmt.Errorf("cannot obtain bus name '%s'", busName)
-		return err
-	}
+		if reply != dbus.RequestNameReplyPrimaryOwner {
+			err = fmt.Errorf("cannot obtain bus name '%s'", iface.Name())
+			return err
```

Comment:
These might be a single line. Nitpick, clearly.. no need to touch it if that's the only change.